README.md

```diff
@@ -82,14 +82,15 @@ import.setConfig({
 })
 ```
 
-Works for any Roblox instance, so you can use this to import assets as well:
+You can configure the alias which you use to represent `script` in your paths.
 
 ```lua
--- local sound = script.Parent:FindFirstChild("Sound")
-local sound = import "./Sound"
+import.setConfig({
+	currentScriptAlias = "@"
+})
 
--- local sound = script.Parent:FindFirstChild("Part")
-local part = import "./Part"
+-- local module = require(script.Module)
+local module = import "@/Module"
```

Comment:
One quick comment about this, does the importer no longer support instances of other types now?

Reply:
oh no it does, i moved around the config MD

Reply:
oops i messed it up, will put that back